roles/sap_ha_cluster/tasks/preconfigure_platform_aws.yml

```diff
@@ -0,0 +1,62 @@
+---
+# Requirement to enable the fencing resource to function.
+
+- name: "SAP HA Cluster - Create awscli config directory"
+  ansible.builtin.file:
+    mode: 0755
+    owner: root
+    path: /root/.aws
+    state: directory
+
+- name: "SAP HA Cluster - Create awscli config"
+  ansible.builtin.blockinfile:
+    backup: true
+    block: |
+      [default]
+      region = {{ sap_ha_cluster_aws_region }}
+    create: true
+    mode: 0600
+    owner: root
+    path: /root/.aws/config
+
+- name: "SAP HA Cluster - Store awscli credentials"
+  ansible.builtin.blockinfile:
+    backup: true
+    block: |
+      [default]
+      aws_access_key_id = {{ sap_ha_cluster_aws_access_key_id }}
+      aws_secret_access_key = {{ sap_ha_cluster_aws_secret_access_key }}
+    create: true
+    mode: 0600
+    owner: root
+    path: /root/.aws/credentials
```

Comment:
no_log: true
please

Reply:
Thanks for the hint! Added in https://github.com/sap-linuxlab/community.sap_install/pull/210/commits/80342f2f0c9fab5213058703bf6e11d6f80db399